src/_image_wrapper.cpp

```diff
@@ -17,39 +17,36 @@ const char* image_resample__doc__ =
 
 "Parameters\n"
 "----------\n"
-"input_array : 2-d or 3-d Numpy array of float, double or uint8\n"
+"input_array : 2-d or 3-d array of float, double or uint8\n"
 "    If 2-d, the image is grayscale.  If 3-d, the image must be of size\n"
 "    4 in the last dimension and represents RGBA data.\n\n"
 
-"output_array : 2-d or 3-d Numpy array of float, double or uint8\n"
-"    The dtype and number of dimensions must match `input_array`.\n\n"
+"output_array : 2-d or 3-d array of float, double or uint8\n"
```

Comment:
these should probably stay to be explicit?

Reply:
Sure; input_array can be any array-like, though (it goes through PyArray_FromAny); edited accordingly.